# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/cairo/hotfix-mutex.patch.darwin
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- cairo-1.16.0/src/cairo-mutex-impl-private.h.vanilla	2020-11-28 13:48:40.000000000 +0100
+++ cairo-1.16.0/src/cairo-mutex-impl-private.h	2020-11-28 13:49:15.000000000 +0100
@@ -156,6 +156,8 @@
  *   poke around cairo-mutex-private.h for possible solutions.
  */
 
+#define CAIRO_NO_MUTEX 1
+
 #if CAIRO_NO_MUTEX
 
 /* No mutexes */
